include/psa/crypto_values.h

```diff
@@ -373,7 +373,7 @@
  */
 #define PSA_KEY_TYPE_DERIVE                     ((psa_key_type_t)0x52000000)
 
-/** Key for an cipher, AEAD or MAC algorithm based on the AES block cipher.
+/** Key for a cipher, AEAD or MAC algorithm based on the AES block cipher.
```

Comment:
Commit message "Grammar" should be "Fix grammar"

Reply:
I don't see why, but ok, whatever.

Reply:
Gotta start with the verb, man.